babel/numbers.py

```diff
@@ -701,8 +719,8 @@ def _format_int(self, value, min, max, locale):
             gsize = self.grouping[1]
         return value + ret
 
-    def _format_frac(self, value, locale):
-        min, max = self.frac_prec
+    def _format_frac(self, value, locale, force_frac=None):
+        min, max = force_frac or self.frac_prec
```

Comment:
Shadowing builtins is just fine.  Babel already does that in many places.  Please do not change this now because that will just make the API inconsistent.  There is nothing wrong with shadowing builtins as you can easily rebind them if you need them locally.